lib/tss2-esys.pc.in

```diff
@@ -10,4 +10,4 @@ Version: @VERSION@
 Requires.private: tss2-mu tss2-sys
 Cflags: -I${includedir}
 Libs: -ltss2-esys -L${libdir}
-Libs.private: @LIBDL_LDFLAGS@ @LIBSOCKET_LDFLAGS@ @TSS2_ESYS_LDFLAGS_CRYPTO@
+Libs.private: @LIBADD_DL@ @LIBSOCKET_LDFLAGS@ @TSS2_ESYS_LDFLAGS_CRYPTO@
```

Comment:
Don't we need  AC_LIB_LTDL or something similar in configure.ac that would configure substitution of this macro?

Reply:
It needs [`LT_LIB_DLLOAD` in `configure.ac`,](https://github.com/tpm2-software/tpm2-tss/blob/15073cc305da6e378ea14aa69cb37f273a1ace39/configure.ac#L33) which was added in #1457 to replace the previous manual approach by the `LIBADD_DL` output variable from [`ltdl.m4`](https://github.com/autotools-mirror/libtool/blob/b9b44533fbf7c7752ffd255c3d09cc360e24183b/m4/ltdl.m4#L741). The new variable is already used successfully in the [Makefile](https://github.com/tpm2-software/tpm2-tss/blob/9df761e1a7258c51e06cac7ee18c4398693fc937/Makefile.am#L278), which shows that output substitution is working.